Makefile

```diff
@@ -157,6 +157,17 @@ delete_minikube_profile:
 delete_podman_localhost:
 	make -C assisted-service/ clean-onprem || true
 
+####################
+# Load balancer    #
+####################
+
+start_load_balancer:
+	# Start load balancer if it does not already exist.  Map the directory $(HOME)/.test-infra/etc/nginx/stream.d to be /etc/nginx/stream.d so it will be used by the python code to fill up load balancing definitions
+	id=`podman ps --quiet --filter "name=load_balancer"` ; ( test -z "$$id" && podman run -d --rm --net=host --name=load_balancer -v $(HOME)/.test-infra/etc/nginx/stream.d:/etc/nginx/stream.d load_balancer:latest ) || ! test -z "$$id"
```

Comment:
what is registry? docker io?

Reply:
we will need this image inside http://quay.io/

Reply:
Local.  We build it locally